openshift/template-prod.yaml

```diff
@@ -41,7 +41,10 @@ objects:
           - name: AWS_S3_ENDPOINT_URL
             value: ""
           - name: AWS_S3_BUCKET_NAME
-            value: ${S3_BUCKET_NAME}
+            valueFrom:
+              secretKeyRef:
+                key: hpf-maven-insights-s3
+                name: bucket
           - name: MODEL_VERSION
```

Comment:
We need the changes to access keys here as well.

Reply:
Done